### PR TITLE
closes #8 clarify company shared IP owners

### LINKED ISSUE #8: Update the Apache 2.0 license to match the co-dev companies involved
### Background Information

Once we know exactly which companies are committed

### Definition of Done

_No response_

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by zdz2101 — COMMENTED** [review]
Based on monday's call, were we waiting on thomas' response to this as well?

**Comment by rossfarrugia** [comment]
> Based on monday's call, were we waiting on thomas' response to this as well?

Yes good point, guess no harm to give it a week or 2 to hear and then we can merge this one later. FYI @thomas-neitmann

File: LICENSE.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ recommend that a file or class name and description of purpose be included on
 the same “printed page” as the copyright notice for easier identification within
 third-party archives.
 
-    Copyright 2021 F. Hoffmann-La Roche AG and GlaxoSmithKline LLC
+    Copyright 2024 F. Hoffmann-La Roche AG, GlaxoSmithKline LLC, Pfizer and Cytel
```
Comment (zdz2101) [inline LICENSE.md:182]:
```suggestion
    Copyright 2024 F. Hoffmann-La Roche AG, GSK plc, Pfizer and Cytel
```

Totally forgot but I recall they changed their name and moving forward it should be this?

Comment (rossfarrugia) [reply LICENSE.md]: A quick Google seems to say you are correct 😄 FYI @bms63 in case this needs to be updated in any other LICENSE files.

Comment (bms63) [reply LICENSE.md]: This was discussed a while back, I think with @fshanlee, and it was decided to leave as GlaxoSmithKline LLC, as this is the legal entity??

Comment (bms63) [reply LICENSE.md]: Maybe @MichaelRimler also discussed this previously??

Comment (rossfarrugia) [reply LICENSE.md]: ah ok, let me put back how it was and can always update later as you need

**Comment by rossfarrugia** [comment]
@zdz2101 fine to approve and merge this one now
